efficientdet/dataset/inspect_tfrecords.py

```diff
@@ -81,7 +77,7 @@ def visualize(self):
     scale_image = tf.constant([0.229, 0.224, 0.225])
     scale_image = tf.reshape(scale_image, (1, 1, -1))
 
-    logging.info(f"Visualizing TfRecords {FLAGS.file_pattern} ....")
+    logging.info('Visualizing TfRecords %s', FLAGS.file_pattern)
```

Comment:
@mingxingtan i thought using f strings is better way as of python >=3.6

Reply:
Hi @kartik4949, yes, f-strings are generally good, but logging prefers %s for the benefit of lazy evaluation.

f-string would cause pylint warning logging-format-interpolation, see: https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation?rq=1

Reply:
Informative :) thanks